components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/CodeTokenResponseValidator.java

```diff
@@ -0,0 +1,66 @@
+package org.wso2.carbon.identity.oauth.common;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.oltu.oauth2.as.validator.TokenValidator;
+import org.apache.oltu.oauth2.common.OAuth;
+import org.apache.oltu.oauth2.common.error.OAuthError;
+import org.apache.oltu.oauth2.common.exception.OAuthProblemException;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.wso2.carbon.identity.oauth.common.OAuthConstants.OAuth20Params.NONCE;
+import static org.wso2.carbon.identity.oauth.common.OAuthConstants.OAuth20Params.SCOPE;
+
+public class CodeTokenResponseValidator extends TokenValidator {
+
+    public CodeTokenResponseValidator() {
+    }
+
+    public void validateRequiredParameters(HttpServletRequest request) throws OAuthProblemException {
```

Comment:
Add method level comment

Reply:
This is actually an overridden method. Is it necessary to add a method comment?